### PR TITLE
fix: #10232 Improved italian translations

### LINKED ISSUE #10232: Italian translation
Please can you translate all the menù on the right click?
Now he is half Italian and half English, a bit annoying.

Thank you,
Simone
dbeaver

### PR DESCRIPTION
**Pull request by acoluzzi**
I've tried to fill in as much tokens as I could while resolving #10232

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ui.editors.data/OSGI-INF/l10n/bundle_it.properties
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.ui.editors.data/OSGI-INF/l10n/bundle_it.properties]:
```suggestion
command.org.jkiss.dbeaver.ui.editors.data.forSelection.name=Mostra dati
```

**Comment by LonwoLonwo** [comment]
Thanks for your contribution.
Looks nice.
Waiting for one more review and QA Team.

**Review by LonwoLonwo — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Comment by acoluzzi** [comment]
Hi guys, Can this be merged by any case?

**Comment by ShadelessFox** [comment]
This PR awaits our QA team.

**Comment by MashaKorax** [comment]
verified

**Pull request merged**
